submissions/krylenger/dom-api-task/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Motorcycles</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <input id='nav__toggle' type="checkbox" class="nav__toggle">
+    <label for="nav__toggle" class="nav__toggle-label"></label>
+    <nav class="nav">
+        <ul class="nav__ul">
+            <li class="nav__li"><a class="nav__link">ADVENTURE</a></li>
+            <li class="nav__li"><a class="nav__link">NAKED</a></li>
+            <li class="nav__li"><a class="nav__link">CRUISER</a></li>
+            <li class="nav__li"><a class="nav__link">TOURING</a></li>
+            <li class="nav__li"><a class="nav__link">SPORT</a></li>
+            <li class="nav__li"><a class="nav__link">SUPERMOTO</a></li>
+            <li class="nav__li"><a class="nav__link">OFFROAD</a></li>
+            <li class="nav__li"><a class="nav__link">RETRO</a></li>
+        </ul>
+    </nav>
+    <main class="main">
+        <h1 class="main__header"> Choose your bike</h1>
+        <div class="main__img-container">
```

Comment:
Is this className written according to the BEM principles? (This questions spreads to another classnames as well)

Reply:
Sorry, I can't see where is the mistake. 
Could you please point to the line number? 
You mean line 29? 
But according to BEM:  block__it-just-the-name-of-element. 
And we can add as many '-' as we want in the element name. They mean nothing and just help us to write element's name more explicit.
Or you mean something else?

Reply:
Sorry, it's my fault. I thought you use `-container` for modificator. 
Gotcha!